charts/umbrella/values-tls.yaml

```
@@ -0,0 +1,566 @@
+              dataconsumerTwo:
+                connectorUrl: "https://dataconsumer-2-controlplane.tx.test/api/v1/dsp"
+              company4:
+                connectorUrl: "https://company4-controlplane.tx.test/api/v1/dsp"
+              company5:
+                connectorUrl: "https://company5-controlplane.tx.test/api/v1/dsp"
+              company6:
+                connectorUrl: "https://company6-controlplane.tx.test/api/v1/dsp"
+              company7:
+                connectorUrl: "https://company7-controlplane.tx.test/api/v1/dsp"
+              company8:
+                connectorUrl: "https://company8-controlplane.tx.test/api/v1/dsp"
+              company9:
+                connectorUrl: "https://company9-controlplane.tx.test/api/v1/dsp"
+              company10:
+                connectorUrl: "https://company10-controlplane.tx.test/api/v1/dsp"
+              company11:
+                connectorUrl: "https://company11-controlplane.tx.test/api/v1/dsp"
+              company12:
+                connectorUrl: "https://company12-controlplane.tx.test/api/v1/dsp"
+              company13:
+                connectorUrl: "https://company13-controlplane.tx.test/api/v1/dsp"
+              company14:
+                connectorUrl: "https://company14-controlplane.tx.test/api/v1/dsp"
+              company15:
+                connectorUrl: "https://company15-controlplane.tx.test/api/v1/dsp"
+              company16:
+                connectorUrl: "https://company16-controlplane.tx.test/api/v1/dsp"
+    ingress:
+      annotations:
+        # uncomment the following line for tls
+        cert-manager.io/issuer: "umbrella-ca-issuer"
+        nginx.ingress.kubernetes.io/use-regex: "true"
+        nginx.ingress.kubernetes.io/enable-cors: "true"
+        nginx.ingress.kubernetes.io/proxy-body-size: "8m"
+        nginx.ingress.kubernetes.io/cors-allow-origin: "http://localhost:3000, https://*.tx.test"
+      tls:
+        # -- Provide tls secret.
+        - secretName: "portal-backend.tx.test-tls"
+        # -- Provide host for tls secret.
+          hosts:
+            - "portal-backend.tx.test"
+
+centralidp:
+  keycloak:
+    ingress:
+      annotations:
+        # uncomment the following line for tls
+        cert-manager.io/issuer: "umbrella-ca-issuer"
+        nginx.ingress.kubernetes.io/cors-allow-credentials: "true"
+        nginx.ingress.kubernetes.io/cors-allow-methods: "PUT, GET, POST, OPTIONS"
+        nginx.ingress.kubernetes.io/cors-allow-origin: "http://centralidp.tx.test"
+        nginx.ingress.kubernetes.io/enable-cors: "true"
+        nginx.ingress.kubernetes.io/proxy-buffer-size: "128k"
+        nginx.ingress.kubernetes.io/proxy-buffering: "on"
+        nginx.ingress.kubernetes.io/proxy-buffers-number: "20"
+        nginx.ingress.kubernetes.io/use-regex: "true"
+      tls: true
+      hostname: "centralidp.tx.test"
+  realmSeeding:
+    sslRequired: "none"
```

As recommended in [portal-iam](https://github.com/eclipse-tractusx/portal-iam/blob/2a38b1c5fac7bb93e45df3da00d56154a349d6e6/charts/sharedidp/values.yaml#L158):
```suggestion
      # 'none' should only be set in non-productive scenarios.
      sslRequired: external
```